sstable/README.md

```diff
@@ -46,9 +46,10 @@ Overview of the SSTable format. Unless noted otherwise, numbers are little-endia
 +----------+--------+-------+-------+-----+
 | BlockLen | Values | Delta | Delta | ... |
 +----------+--------+-------+-------+-----+
-                    |----( # of deltas)---|
+           |        |----( # of deltas)---|
+           |------(maybe compressed)------|
 ```
-- BlockLen(u32): length of the block
+- BlockLen(u32): length of the block. If MSB is 1, block is compressed with zstd, otherwise it's not.
```

Comment:
I'm not fond of the single bit trick here.
Can we have a byte in the begining instead. The bit solution will make it painful if we want to change the compression format.